pipeline/extract.py

```diff
@@ -1,59 +1,46 @@
 """Extracts all the plant data from plant 0 - 50 and adds to csv file."""
 
 import os
+import time
 import requests
 import pandas as pd
```

Comment:
https://peps.python.org/pep-0008/#imports 

Imports have a specific order suggested in pep-8:


    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

with a blank line between each group 
```suggestion
import os
import requests
import time

import pandas as pd
```